Yoast/wordpress-seo
# Hides the upsell for non supported (word form recognition) languages

## Summary

This PR can be summarized in the following changelog entry:

* N/A


## Test instructions

This PR can be tested by following these steps:

* Set your site to a non-English language.
* Checkout `release/9.0` and open the metabox for a post.
* See that there is an upsell under the keyword phrase field.
* Go to the term page and verify there isn't an upsell
* Checkout this branch and do the `grunt build` magic
* The upsell under the keyword phrase field is gone.
* Set the site language to English
* Go to the term page and verify that there is an upsell.

## Quality assurance


Fixes #11282
